controllers/app.controllers.js

```diff
@@ -0,0 +1,11 @@
+const { selectAllTopics } = require("../models/app.models");
+
+exports.getApiTopics = (req, res) => {
+  selectAllTopics()
+    .then(({rows}) => {
+      res.status(200).send({ topics: rows });
+    })
+    .catch((err) => {
+      console.log(err);
+    });
```

Comment:
You havent done any error handling for this, have you written any tests that are testing this?